### PR TITLE
add `release.yml`

### PR DESCRIPTION
**Pull request by B2o5T**

### CODE REVIEW

File: .github/workflows/canary.yml [hunk 1]
```diff
@@ -33,10 +33,10 @@ jobs:
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
-          key: ${{runner.os}}-16-8-16-node-modules-${{hashFiles('yarn.lock')}}
+          key: ${{runner.os}}-yarn-${{hashFiles('yarn.lock')}}
           restore-keys: |
-            ${{runner.os}}-16-8-16-node-modules-${{hashFiles('yarn.lock')}}
-            ${{runner.os}}-16-8-16-node-modules-
```
Comment (ardatan) [inline .github/workflows/canary.yml:39]: I think we should respect versions/variations on tests (for Nodejs version matrix etc)

Comment (B2o5T) [reply .github/workflows/canary.yml:39]: yeah, will change

**Comment by Urigo** [comment]
should/can we reuse workflows across repos?

**Comment by B2o5T** [comment]
> should/can we reuse workflows across repos?

@Urigo I think this is good point, we must reuse `canary`, `release` and `github` workflow, there are the same across repos, but need to keep in mind that env variables must be the same also

**Pull request merged**